simulator/src/output_settings.rs

```diff
@@ -31,11 +31,13 @@ impl OutputSettings {
     }
 
     /// Translates a output coordinate to the corresponding display coordinate.
+    #[allow(unused)]
```

Comment:
Would it be cleaner to move these two functions into a second `impl OutputSettings` block and use the `#[cfg(feature = "with-sdl")]` on that block?

Reply:
Ah yes, I forgot you can have multiple impl blocks, thanks!